src/components/Form/СreateUserAccountForm/CreateUserAccountForm.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useMemo, useState } from 'react';
+import { Box, TextField, Typography } from '@material-ui/core';
+import i18n from 'i18next';
+import { langTokens } from '../../../locales/localizationInit';
+import { UserAccountButton } from '../Buttons';
+import { useStyles } from './styles/CreateUserAccountForm.style';
+import { regex } from '../../../views/Profile/PersonalInfo/constants/regex';
+import { UserEmailType } from '../../../views/Profile/PersonalInfo/types';
+
+type CreateUserAccountFormPropsType = {
+  usersEmails: UserEmailType;
+  onClick: (
+    e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+    email: string,
+  ) => void;
+  currentUserEmail?: string;
+  isLoading?: boolean;
+};
+type ErrorFieldsType = {
+  email: string;
+};
+type VisitedFieldsType = {
+  email: boolean;
+};
+export const CreateUserAccountForm: React.FC<CreateUserAccountFormPropsType> = ({
+  onClick,
+  usersEmails,
+  currentUserEmail,
+  isLoading,
+}) => {
+  const [userEmail, setUserEmail] = useState('');
+  const [visitFields, setVisitFields] = useState<VisitedFieldsType>({
+    email: false,
+  });
+
+  const classes = useStyles();
+  const emailChangeHandler = (
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+  ) => {
+    setUserEmail(e.target.value);
+  };
+
+  const errorMessage: ErrorFieldsType = useMemo(() => {
+    const errors: ErrorFieldsType = {
+      email: '',
+    };
+
+    if (currentUserEmail) {
+      errors.email = i18n.t(langTokens.admin.emailConfirmationWasSent);
+    } else if (!userEmail && visitFields.email) {
+      errors.email = i18n.t(langTokens.admin.fieldCantBeEmpty);
+    } else if (userEmail && !regex.validEmail.test(userEmail)) {
+      errors.email = i18n.t(langTokens.admin.wrongEmail);
+    } else if (
+      userEmail &&
+      usersEmails.publicEmail.some((mail) => mail === userEmail)
+    ) {
+      errors.email = i18n.t(langTokens.admin.diffEmail);
+    } else if (!userEmail && !visitFields.email) {
+      errors.email = ' ';
```

Comment:
so there is un-empty error message but it's not visible to the user?

Reply:
Just to disable button to show user that he need to input email